exampleCourse/elements/clickable-image/clickable-image.py

```diff
@@ -84,3 +89,12 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
         "weight": weight,
         "feedback": feedback,
     }
+
+
+def test(element_html: str, data: pl.ElementTestData) -> None:
+    element = lxml.html.fragment_fromstring(element_html)
+    name = pl.get_string_attrib(element, "answers-name")
+    if data["test_type"] != "correct":
+        raise NotImplementedError
```

Comment:
It seems like it wouldn't be too hard to actually define incorrect/invalid submissions here?

Reply:
Changed in [08ec836](https://github.com/PrairieLearn/PrairieLearn/pull/11907/commits/08ec83674631cfbf692eb050cfe13573f9c96c46).